tests/unit/s2n_handshake_io_early_data_test.c

```diff
@@ -28,12 +26,31 @@ int main(int argc, char **argv)
     BEGIN_TEST();
 
     uint8_t bad_record[] = {
-            0x17, /* ContentType opaque_type = application_data */
-            0x03, 0x03, /* ProtocolVersion legacy_record_version = 0x0303 */
-            0x00, 0x10, /* uint16 length */
-            /* opaque encrypted_record[TLSCiphertext.length] */
-            0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
-            0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F,
+        /* ContentType opaque_type = application_data */
+        0x17,
+        /* ProtocolVersion legacy_record_version = 0x0303 */
+        0x03,
+        0x03,
+        /* uint16 length */
+        0x00,
+        0x10,
+        /* opaque encrypted_record[TLSCiphertext.length] */
+        0x00,
```

Comment:
Heh can we manually fix this one. This is just really weird. The original formatting is fine.

Reply:
Sure! Would you like the comments after the data or should I use clang-format off to restore the original formatting exactly?

Reply:
Change the comments to be after the data please